stanovy_vpsFree.cz.md

```diff
@@ -305,107 +368,104 @@ systému Spolku nejméně 10 pracovních dnů před začátkem platnosti.
+  členům a veřejným oznámením v informačním systému Spolku,
   7. svolává jednání Rady pokud Rada nesvolá řádné zasedání podle §17,
   8. svolává neprodleně mimořádné jednání Schůze členů, pokud Schůzi členů
-nesvolá Rada v řádném termínu, nebo pokud to vyžadují zájmy Spolku.
+  nesvolá Rada v řádném termínu, nebo pokud to vyžadují zájmy Spolku
 4. Rada je povinna umožnit Kontrolní komisi veřejné a neprodlené publikování
-zpráv a plnění povinností vyplývající ze stanov.
+zpráv a plnění povinností vyplývající ze stanov
 5. Kontrolní komise je povinna upozornit Radu a členy oznámením ve formě
 elektronické zprávy zaslané všem členům Spolku a veřejným oznámením
 v informačním systému Spolku na porušení zákonů, stanov, na nehospodárné
 postupy, popřípadě na další nedostatky v činnosti Spolku.
 
 ### §22 Složení a funkční období
 
-1. Kontrolní komise má 3 členy.
-2. Členy Kontrolní komise volí Schůze členů.
+1. Kontrolní komise má 3 členy
+2. Členy Kontrolní komise volí Schůze členů
 3. Členem Kontrolní komise může být pouze fyzická osoba, která dosáhla věku
 18 let, která je plně způsobilá k právním úkonům, je členem Spolku a není ani
-sama ani osoby jí blízké se Spolkum v pracovněprávním nebo jiném obdobném
+sama ani osoby jí blízké se Spolkem v pracovněprávním nebo jiném obdobném
 vztahu. Za bezúhonnou se považuje osoba, která je bezúhonná ve smyslu zákona
-o živnostenském podnikání.
-4. Členství v Kontrolní komisi je neslučitelné s členstvím v Radě.
-5. Funkční období člena Kontrolní komise není omezeno časově. Člen Kontrolní
+o živnostenském podnikání
+4. Členství v Kontrolní komisi je neslučitelné s členstvím v Radě
+5. Funkční období člena Kontrolní komise není omezeno časově; člen Kontrolní
 komise může být odvolán pouze Schůzí členů na jejím pravidelném nebo mimořádném
 zasedání. Výkon funkce člena Kontrolní komise skončí dnem zvolení nového člena
-Kontrolní komise.
-6. Člen Kontrolní komise, který přestal splňovat podmínky uvedené v §23 odst. 3
+Kontrolní komise
+6. Člen Kontrolní komise, který přestal splňovat podmínky uvedené v bodu 3. §23
 nebo odstoupil, je povinen tuto skutečnost oznámit písemným prohlášením
-doručeným Radě. Funkce zaniká dnem doručení takového oznámení.
+doručeným Radě. Funkce zaniká dnem doručení takového oznámení
 7. Kontrolní komise, pokud počet jejích členů zvolených Schůzí členů neklesl
 pod polovinu, může kooptovat náhradní členy Kontrolní komise do konání 
 následující Schůze členů, která zvolí nové členy Kontrolní komise. Kontrolní
 komise musí o úmyslu kooptovat nového člena informovat všechny členy Spolku
 a poskytnout jim 7denní lhůtu na podání kandidatury pro kooptaci. Kontrolní
-komise volí kooptovaného člena z těchto kandidatur.
+komise volí kooptovaného člena z těchto kandidatur
 8. Členům Kontrolní komise může příslušet za výkon jejich funkce odměna. O výši
 a způsobu výplaty odměny rozhoduje Rada Spolku. Členové Kontrolní komise mají
```

Comment:
návrh: KK by měl být orgán zodpovědný členům (resp. členské schůzi). O případné odměně měla rozhodovat schůze a nikoliv rada. Takto odměny radě schvaluje KK a odměny KK schvaluje rada. Zde vidím potenciální závislost.

Reply:
taky overkill. Tahle změna by měla být zaměřená jen na legal cleanup, dálkový hlasování  a vedlejší činnost (tj. věci, který se probíraly na schůzi).

tohle bych klidně nechal na diskusi někdy do budoucna. Jakmile budeme moct hlasovat dálkově, nebude už problém ty stanovy doplnit kdykoliv v budoucnu.

Reply:
OK, tohle je fakt na diskusi a jen by to zdržovalo.